ssh-uuid.sh

```diff
@@ -367,8 +367,11 @@ function do_proxy {
 	{ set +x; } 2>/dev/null
 	sleep 1 # poor man's wait for the background socat process to be ready
 	[ -n "${DEBUG}" ] && set -x
+	set +e
```

Comment:
```suggestion
	set +e
	[ -n "${DEBUG}" ] && set -x
```
I suggest moving the `set +e` line before the `set -x` line (move it one line up) so that `set +e` does not clutter debugging output (although, to be honest, I don’t remember where this debug output gets printed, given that the `do_proxy` function is only executed by the `ssh` process).

(Generally, my intention for `set -x` is for command lines “of interest” to be printed in debugging output. I consider the `socat` command line to be of interest, but not the `set +e` command line. In this script file, you will find that `set -x` immediately precedes command lines like `scp`, `ssh`, `socat`, `balena-engine`...)